docs/src/app/components/pages/page-with-nav.jsx

```diff
@@ -68,30 +73,34 @@ let PageWithNav = React.createClass({
         <div style={this.prepareStyles(styles.content)}>
           {this.props.children}
         </div>
-        <div style={this.prepareStyles(styles.secondaryNav)}>
-          <Menu
-            ref="menuItems"
-            zDepth={0}
-            menuItems={this.props.menuItems}
-            selectedIndex={this._getSelectedIndex()}
-            onItemTap={this._onMenuItemClick} />
-        </div>
+        <Menu
+          autoWidth={false}
+          onItemTouchTap={this._handleItemTouchTap}
+          openDirection="bottom-right"
+          style={styles.secondaryNav}
+          valueLink={{
```

Comment:
I didn't know `valueLink` was being deprecated...I'm not a huge fan of it anyway :P

Was just implementing it the easiest way I saw possible from the menu docs, I'll find another way.